New log file opened at Fri Oct 30 21:16:57 CET 2015 by 3316@Toshiba-Robin
Taking next connection to: 50094
New client attempting to connect on port: 50095-50094
  Address = /192.168.178.12:50094
This client on port 50095 has uuid 5964484c-f7be-4d3c-9a18-48ae934fd69f configName activator humanReadableName Activator protocolVersion 3 featureTags Vector()
Connected Clients: activator-5964484c-f7be-4d3c-9a18-48ae934fd69f
Taking next connection to: 50094
New client attempting to connect on port: 50096-50094
  Address = /192.168.178.12:50094
Handshake exception on socket: 50096
sbt.impl.ipc.HandshakeException: Unable to perform handshake
	at sbt.impl.ipc.Peer.handshake(IPC.scala:46)
	at sbt.impl.ipc.MultiClientServer.<init>(IPC.scala:150)
	at sbt.server.SbtServerSocketHandler$$anon$1.run(SbtServerSocketHandler.scala:43)
Caused by: java.net.SocketException: Software caused connection abort: recv failed
	at java.net.SocketInputStream.socketRead0(Native Method)
	at java.net.SocketInputStream.socketRead(Unknown Source)
	at java.net.SocketInputStream.read(Unknown Source)
	at java.net.SocketInputStream.read(Unknown Source)
	at java.io.BufferedInputStream.fill(Unknown Source)
	at java.io.BufferedInputStream.read(Unknown Source)
	at java.io.DataInputStream.readInt(Unknown Source)
	at sbt.impl.ipc.Peer.receive(IPC.scala:95)
	at sbt.impl.ipc.Peer.handshake(IPC.scala:49)
	... 2 more
Taking next connection to: 50094
New client attempting to connect on port: 50097-50094
  Address = /192.168.178.12:50094
This client on port 50097 has uuid daec09a1-5c28-40e9-92d4-7a8c851589e0 configName activator humanReadableName Activator protocolVersion 3 featureTags Vector()
Connected Clients: activator-5964484c-f7be-4d3c-9a18-48ae934fd69f, activator-daec09a1-5c28-40e9-92d4-7a8c851589e0
Taking next connection to: 50094
New client attempting to connect on port: 50107-50094
  Address = /192.168.178.12:50094
Handshake exception on socket: 50107
sbt.impl.ipc.HandshakeException: Unable to perform handshake
	at sbt.impl.ipc.Peer.handshake(IPC.scala:46)
	at sbt.impl.ipc.MultiClientServer.<init>(IPC.scala:150)
	at sbt.server.SbtServerSocketHandler$$anon$1.run(SbtServerSocketHandler.scala:43)
Caused by: java.net.SocketException: Software caused connection abort: recv failed
	at java.net.SocketInputStream.socketRead0(Native Method)
	at java.net.SocketInputStream.socketRead(Unknown Source)
	at java.net.SocketInputStream.read(Unknown Source)
	at java.net.SocketInputStream.read(Unknown Source)
	at java.io.BufferedInputStream.fill(Unknown Source)
	at java.io.BufferedInputStream.read(Unknown Source)
	at java.io.DataInputStream.readInt(Unknown Source)
	at sbt.impl.ipc.Peer.receive(IPC.scala:95)
	at sbt.impl.ipc.Peer.handshake(IPC.scala:49)
	... 2 more
Taking next connection to: 50094
New client attempting to connect on port: 50108-50094
  Address = /192.168.178.12:50094
This client on port 50108 has uuid 12ce5a5a-298c-41d9-87f0-11972bf7f761 configName play-fork-run humanReadableName Play Fork Run protocolVersion 1 featureTags Vector()
Connected Clients: activator-daec09a1-5c28-40e9-92d4-7a8c851589e0, play-fork-run-12ce5a5a-298c-41d9-87f0-11972bf7f761
Taking next connection to: 50094
Checking to see if clients are empty...
We have 1 non-daemon clients, continuing serving connections.
Taking next connection to: 50094
Checking to see if clients are empty...
We have 1 non-daemon clients, continuing serving connections.
Taking next connection to: 50094
Checking to see if clients are empty...
We have 1 non-daemon clients, continuing serving connections.
Taking next connection to: 50094
Checking to see if clients are empty...
No non-daemon clients connected after 3 min. Shutting down.
1 daemon clients are connected.
Server socket thread exiting.
1 clients open, will close them...
All client sockets have been closed.
Closing the 3316@Toshiba-Robin logs at Fri Oct 30 21:31:04 CET 2015, goodbye.
